pkg/square/testdata/fuzz/FuzzSquare/dd4df2aa7925a6a2

```diff
@@ -0,0 +1,5 @@
+go test fuzz v1
```

Comment:
[question] should this file be checked into version control?

Reply:
No it must have slipped in. I'll remove it